src/etc/inc/gwlb.inc

```diff
@@ -1343,7 +1343,12 @@ function get_gwgroup_members_inner($group, $gateways_status, $gateways_arr, $vip
 						$groupmember = array();
 						$groupmember['gw'] = $member;
 						$groupmember['int'] = $int;
-						$groupmember['gwip'] = $gatewayip;
+						if(is_ipaddrv6($gatewayip) && is_linklocal($gatewayip))
+						{
+							$groupmember['gwip'] = $gatewayip + '%' + $int;
```

Comment:
`$int` as you have it here is the internal name of the interface (e.g. `lan`, `wan`, `opt4`) and not what you need, which is the OS level interface (e.g. `igb0`, `ix2`, `igc3`).

You need to run it through something like `get_real_interface($int)` and sanity check the output before using it.

Reply:
I do not believe this is the case.  I added log lines to be sure, and the value of `$int` at this time is `igb0` for me.

Reply:
You are right, I was looking at the field name in the configuration but this section of code retrieved the gateway another way which contains the right value there.